example/lib/common_blocs/simple_linked_accounts_bloc/simple_linked_accounts_bloc.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:bloc_concurrency/bloc_concurrency.dart';
+import 'package:domain/accounts/repository/accounts_repository.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:kevin_flutter_example/common_blocs/simple_linked_accounts_bloc/simple_linked_accounts_event.dart';
+import 'package:kevin_flutter_example/common_blocs/simple_linked_accounts_bloc/simple_linked_accounts_state.dart';
+
+class SimpleLinkedAccountsBloc
```

Comment:
No plans for that 😄 Name is meant to tell, that bloc is used for super simple (single) action - get linked accounts. I extracted that bloc, because two pages used the same functionality, so there is no need to duplicate code.

Reply:
Will change in one of the upcoming PRs.